### ISSUE TITLE
2019-6-4

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#23]
721.Linear function
722.Sigmoid function
723.1.3 - Computing the Cost
724.Cost Function With sigmoid
725.1.4 - Using One Hot encodings
726.One hot matrix
727.1.5 - Initialize with zeros and ones
728.Ones function
729.2 - Building your first neural network in tensorflow
